corehq/apps/accounting/forms.py

```diff
@@ -160,6 +160,12 @@ class BillingAccountBasicForm(forms.Form):
         help_text="Users in any projects connected to this account will not "
                   "have data sent to Hubspot",
     )
+    bill_web_user = forms.BooleanField(
+        label="Bill Web User",
+        required=False,
+        initial=False,
+        help_text="Make sure this billing account have subscription with Web User Feature"
```

Comment:
nit: repphrase to:

```
"Include Web Users in invoice (requires a subscription with Web User Feature)"
```

Reply:
Rephrased in af28f94056803f93654d0226a93dcd212bb5dfd5. But I'm confused do we really need this. The `help_text` isn't rendered at all, when we use hqcrispy for checkbox. The `help_text` isn't rendered for [block_hubspot_data_for_all_users](https://github.com/dimagi/commcare-hq/blob/jc/bill-web-user-flag/corehq/apps/accounting/forms.py#L156) as well. I searched in our codebase, and find `hqcrispy.FieldWithHelpBubble`, but still cannot make it work.